### ISSUE TITLE
Add option to unpatch

### ISSUE THREAD

**Issue by Camburgaler** [AntonioDePau/KHPCPatchManager#54]
# CURRENT BEHAVIOR
The KHPCPatchManager can patch the games, but the only way to unpatch is to manually locate the backup folder and restore the original copies or to verify game files through Steam.

# PROBLEM STATEMENT
This is inconvenient. The user should be able to restore their games' files to vanilla if they no longer want their game patched.

# TARGET BEHAVIOR
Add a button to "Unpatch" the game(s). Clicking this button restores the patched files to their respective original states based on the backup folder.

**Comment by AntonioDePau** [comment]
Yeah, I mean, the button would basically serve as a shortcut to do File.Move, I guess.
If that's a nice enough QoL improvement, why not.
